lib/gui/modules/drive-scanner.js

```diff
@@ -32,7 +32,7 @@ const driveScanner = angular.module(MODULE_NAME, [
   require('../models/settings')
```

Comment:
If this component doesn't depend on `SettingsModel`  anymore, then it should remove it from its dependencies.